src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectAuthenticationHandler.cs

```diff
@@ -30,6 +32,22 @@ public class OpenIdConnectAuthenticationHandler : AuthenticationHandler<OpenIdCo
     {
         private const string NonceProperty = "N";
         private const string UriSchemeDelimiter = "://";
+
+        private const string InputTagFormat = @"<input type=""hidden"" name=""{0}"" value=""{1}"" />";
```

Comment:
I did, it's used further down at line 132, with encoding happening in the previous 2 lines.